player.go

```diff
@@ -19,7 +19,8 @@ import (
 	"time"
 )
 
-// A Player is a sound player.
+// Player is a PCM (pulse-code modulation) audio player. It implements io.Writer, use Write method
```

Comment:
I think it's very common for struct names

e.g. https://golang.org/pkg/go/ast/#ArrayType